lib/normalizers/v1.js

```diff
@@ -54,9 +54,11 @@ _.assign(Builders.prototype, {
         if (!entityV1) { return; }
         // Handle the noauth/normal case for new and legacy attributes first, in that order
         if (util.notLegacy(entityV1, 'auth') && entityV1.auth) {
-            return (entityV1.auth.type === 'noauth') ? { type: 'noauth' } : util.sanitizeAuthArray(entityV1);
+            return (entityV1.auth.type === 'noauth') ? null : util.sanitizeAuthArray(entityV1);
```

Comment:
I think this shouldn't be the case. `normal` and `null` should be `null`. `noauth` -> `null` conversion should not be done.

Reply:
This is what was discussed in the previous sync up. 😓

Reply:
Consider a v2 collection with `request.auth.type='noauth'`, it gets converted to `request.auth.type='noauth'` in v1. When it gets converted back to v2 it becomes `request.auth.type=null`.

This will result in a data loss.

I'm pretty sure we did not put `noauth` in the inherit bucket in our sync up. 😟

Reply:
@kamalaknn As discussed offline.